packages/nocodb/src/db/functionMappings/commonFns.ts

```diff
@@ -112,4 +111,112 @@ export default {
       builder: (await args.fn(args.pt?.arguments?.[0])).builder.wrap('(', ')'),
     };
   },
+  BLANK: async (args: MapFnArgs) => {
+    return { builder: args.knex.raw(`?`, [null]) };
+  },
+  TRUE: async (args: MapFnArgs) => {
+    return { builder: args.knex.raw(`?`, [1]) };
+  },
+  FALSE: async (args: MapFnArgs) => {
+    return { builder: args.knex.raw(`?`, [0]) };
+  },
```

Comment:
in some db types, checkbox value can be `true` / `false` instead of `1` / `0`. In this case, `IF({checkbox} == TRUE() ,..., ...)` would be always false.

Reply:
We are making it similar to AT, will handle these case in a followup PR.